homework/style.css

```diff
@@ -1,3 +1,107 @@
+* {
```

Comment:
The responsiveness of your app can be improved:

![image](https://user-images.githubusercontent.com/2788771/61944154-4c84ab00-af9d-11e9-9467-dba7aa79b5e8.png)